determined-ai/determined
# chore: Remove chart feature flag, remove unused code [WEB-930]

## Description

- Drop feature flag `f_chart=on`
- LineChart component becomes default on TrialDetails, Profiler, and Multi-Trial LearningCurveChart
- Remove TrialWorkloads table, Step type, and utils only used in TrialWorkloads

## Test Plan

The new chart views have been tested before, so the main test is that they appear by default

- Visit a single-trial experiment without adding a feature flag, and confirm it has a grid of charts
- Visit an experiment with a Profiler ( `experiments/1788` is an example on latest-master ) and confirm it has a tooltip
- Visit a multi-trial experiment and test the learning curve chart (tooltip should have reasonable decimal precision)

## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
